swap_meet/item.py

```diff
@@ -1,2 +1,50 @@
+# from .clothing import Clothing
+# from .decor import Decor 
+# from .electronics import Electronics 
```

Comment:
No need for all of this (and would cause a circular import error if not commented) so best to remove it.